crates/ir/src/container.rs

```diff
@@ -0,0 +1,107 @@
+use std::mem;
+
+use crate::{
+    op_code::{IrInstruction, IrOpCode},
+    register::{chill::RegisterChill, RegisterId},
+};
+
+pub struct IrContainer {
+    _data: Vec<u8>,
```

Comment:
This field is used but its name is prefixed with an underscore. Is that an oversight?